tests/base/base.test.js

```diff
@@ -0,0 +1,15 @@
+import { describe, it, expect } from 'vitest';
+import { ESLint } from 'eslint';
+
+describe('base config', () => {
+  it('should allow more then 3 params passed to function', async () => {
+    // arrange
+    const linter = new ESLint();
+    const fileName = 'tests/base/function-param-test.js';
+
+    // act
+    const [result] = await linter.lintFiles([fileName]);
+
+    expect(result.errorCount).toBe(0);
```

Comment:
The rule warned us and didn't give an error, so we should check for the rule identifier in the messages.

Reply:
makes sense!